databus-client2/src/main/java/com/bazaarvoice/emodb/databus/client2/client/Databus.java

```diff
@@ -0,0 +1,291 @@
+package com.bazaarvoice.emodb.databus.client2.client;
+
+
+import com.bazaarvoice.emodb.databus.client2.exceptions.PollFailedException;
+import com.bazaarvoice.emodb.databus.client2.exceptions.SubscribeFailedException;
+import com.bazaarvoice.emodb.databus.client2.discovery.DatabusDiscovery;
+import com.fasterxml.jackson.core.type.TypeReference;
+
+import org.apache.curator.RetryPolicy;
+import org.apache.curator.framework.CuratorFrameworkFactory;
+import org.apache.curator.retry.RetryOneTime;
+import org.apache.http.HttpStatus;
+import org.glassfish.jersey.client.JerseyClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nullable;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.UriBuilder;
+import java.io.Closeable;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.time.Duration;
+import java.util.List;
+import java.util.Optional;
+
+import static com.bazaarvoice.emodb.databus.client2.Json.JsonUtil.parseJson;
+import static com.bazaarvoice.emodb.databus.client2.Json.JsonUtil.toJsonString;
+
+
+/**
+ * Thin client for accessing EmoDB Databus.
+ */
+public class Databus implements Closeable {
```

Comment:
new Databus implementation should implement `com.bazaarvoice.emodb.databus.api.Databus` interface and all methods.
see https://github.com/bazaarvoice/emodb/blob/master/databus-client-common/src/main/java/com/bazaarvoice/emodb/databus/client/DatabusClient.java as example